extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -199,7 +200,7 @@ private void handleSnappy(BuildProducer<ReflectiveClassBuildItem> reflectiveClas
             String path = root + dir + "/" + snappyNativeLibraryName;
             nativeLibs.produce(new NativeImageResourceBuildItem(path));
         } else { // otherwise the native lib of the platform this build runs on
-            String dir = getOs() + "/" + getArch();
+            String dir = OSInfo.getNativeLibFolderPathForCurrentOS();
```

Comment:
Is getOS and getArch still used?

Reply:
Good you asked :) I hadn't noticed that with this change, these methods will now be unused. I've updated this PR to remove them altogether.